walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -26,26 +26,23 @@
     Collection<String> attendees = request.getAttendees();
     long duration = request.getDuration();
     boolean inattendance = false; 
+    int entireDay = 1440;
     
-    List<Event> newEvents = new ArrayList<Event> ();
+    List<Event> newEvents = new ArrayList<Event>();
 
-    if (duration > 1440)
+    if (duration > entireDay)
     {
         return Arrays.asList();
     }
 
     for (Event event : events) {
-        inattendance = false;  
-        for (String attendee : attendees){ 
+        for (String attendee : attendees) { 
             if (event.getAttendees().contains(attendee))
             {
-                inattendance = true;
+                newEvents.add(event); 
+                break; 
             }
         }
-        if (inattendance)
-        {
-            newEvents.add(event);
-        }
     }
 
     newEvents.sort(Comparator.comparing(Event::getWhen, TimeRange.ORDER_BY_START.thenComparing(TimeRange.ORDER_BY_END)));
```

Comment:
For some reason, Github won't let me comment the line below this, but for 
`List<TimeRange> results = new ArrayList<TimeRange> ();`, there's an extra space before `()`